WordPress/Classes/ViewRelated/Stats/JetpackSettingsViewController.m

```diff
@@ -75,7 +75,7 @@ @implementation JetpackSettingsViewController
 
 - (instancetype)initWithBlog:(Blog *)blog
 {
-    self = [super init];
+    self = [super initWithNibName:nil bundle:nil];
```

Comment:
From the Apple Docs:

```
  The designated initializer. If you subclass UIViewController, you must call the super implementation of this
  method, even if you aren't using a NIB.  (As a convenience, the default init method will do this for you,
  and specify nil for both of this methods arguments.) In the specified NIB, the File's Owner proxy should
  have its class set to your view controller subclass, with the view outlet connected to the main view. If you
  invoke this method with a nil nib name, then this class' -loadView method will attempt to load a NIB whose
  name is the same as your view controller's class. If no such NIB in fact exists then you must either call
  -setView: before -view is invoked, or override the -loadView method to set up your views programatically.
```

I'll switch it back to `init` - for some reason I thought the default init method didn't call the same effective initWithNibName:bundle: with nils.